code/game/objects/structures/beds_chairs/sofa.dm

```diff
@@ -1,67 +1,69 @@
-/obj/structure/chair/sofa
-	name = "old ratty sofa"
-	icon_state = "sofamiddle"
-	icon = 'icons/obj/sofa.dmi'
+/obj/structure/chair/fancy/sofa //like it's brother fancy chair, this is the father of all sofas
+	name = "old father sofa"
+	desc = "Now extint, this kind of sofa shouldn't even exists anymore, if you see this wild specimen, contact your local Nanotransen Anti-couch surfer department."
```

Comment:
Now extinct, this kind of sofa shouldn't even exist anymore. If you see this wild specimen, contact your station's Anti-Couch Surfing Department.